Gemfile

```diff
@@ -71,5 +71,6 @@ group :test do
   gem 'selenium-webdriver'
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Tnx. I will have my logo.